js/controller.js

```diff
@@ -510,6 +510,13 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.resetUpNextInfo(false);
     },
 
+    onAssetDimensionsReceived: function(event, params) {
+      if (params.videoId == OO.VIDEO.MAIN) {
+        this.state.mainVideoAspectRatio = this.calculateAspectRatio(params.width, params.height);
```

Comment:
@rchalooyala, what happens if the aspect ratio is set explicitly in skin config, will we still receive the  ASSET_DIMENSION event and adjust automatically?